RELEASING.md

```diff
@@ -12,7 +12,16 @@ types.
 In the future we might be able to update the build to work
 without having JDK 8 installed, by using the `-release` option.
 
-### MinGW
+### GPG and sbt-pgp setup
+ 
+* Check the [sbt-pgp usage](https://www.scala-sbt.org/sbt-pgp/usage.html) for setup steps you may still need
```

Comment:
Can we explicitly add which steps might be needed? I don't think I had to do anything

Reply:
Curious what, if any GPG you had previously if the sbt plugin just worked for you?

Reply:
Good idea, added.